apps/module-scan/src/store.test.ts

```diff
@@ -43,6 +43,42 @@ test('get/set test mode', async () => {
   expect(await store.getTestMode()).toBe(false)
 })
 
+test('get/set mark threshold overrides', async () => {
+  const store = await Store.memoryStore()
+
+  expect(await store.getMarkThresholdOverrides()).toBe(undefined)
+
+  await store.setMarkThresholdOverrides({ definite: 0.4, marginal: 0.5 })
+  expect(await store.getMarkThresholdOverrides()).toStrictEqual({
+    definite: 0.4,
+    marginal: 0.5,
+  })
+
+  await store.setMarkThresholdOverrides(undefined)
+  expect(await store.getMarkThresholdOverrides()).toBe(undefined)
+})
+
+test('get current mark thresholds falls back to election definition defaults', async () => {
+  const store = await Store.memoryStore()
+  await store.setElection(fromElection(election))
+  expect(await store.getCurrentMarkThresholds()).toStrictEqual({
+    definite: 0.17,
+    marginal: 0.12,
+  })
+
+  await store.setMarkThresholdOverrides({ definite: 0.4, marginal: 0.5 })
+  expect(await store.getCurrentMarkThresholds()).toStrictEqual({
+    definite: 0.4,
+    marginal: 0.5,
+  })
+
+  await store.setMarkThresholdOverrides(undefined)
+  expect(await store.getCurrentMarkThresholds()).toStrictEqual({
```

Comment:
Why `toStrictEqual`?

Reply:
So this lead me to look it up. From the jest docs:

> Use `.toEqual` to compare recursively all properties of object instances (also known as "deep" equality). It calls `Object.is` to compare primitive values, which is even better for testing than `===` strict equality operator.

> Use `.toBe` to compare primitive values or to check referential identity of object instances. It calls `Object.is` to compare values, which is even better for testing than `===` strict equality operator.

> Use `.toStrictEqual` to test that objects have the same types as well as structure.
>
> Differences from .toEqual:
>
> - Keys with undefined properties are checked. e.g. `{a: undefined, b: 2}` does not match `{b: 2}` when using `.toStrictEqual`.
> - Array sparseness is checked. e.g. `[, 1]` does not match `[undefined, 1]` when using `.toStrictEqual`.
> - Object types are checked to be equal. e.g. A class instance with fields `a` and `b` will not equal a literal object with fields `a` and `b`.

This is different from the node `assert` module:

> `deepEqual` tests for deep equality between the actual and expected parameters. Consider using `assert.deepStrictEqual()` instead. `assert.deepEqual()` can have surprising results.

> `strictEqual` tests strict equality between the actual and expected parameters as determined by the SameValue Comparison (`Object.is`).

> `equal` tests shallow, coercive equality between the actual and expected parameters using the Abstract Equality Comparison (` ==` ). `NaN` is special handled and treated as being identical in case both sides are `NaN`.

So `.toEqual` is similar to `deepStrictEqual`, `.toBe` is similar to `strictEqual`, and `.toStrictEqual` is kinda like `deepStrictEqual` but with additional checks. `assert.equal` doesn't really have a jest equivalent, but `.toBe` is probably the most similar. Since it equates to `==` it's mostly not a useful comparison mechanism.